app/human-detector-app/App.tsx

```diff
@@ -2,30 +2,75 @@ import * as React from 'react';
 import { View, Text, StyleSheet, Platform } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { TokenResponse } from 'expo-auth-session';
+import JWT from 'expo-jwt';
+import * as CryptoJS from 'crypto-js';
+import User from './classes/User';
 import CameraScreen from './screens/CameraScreen';
 import GroupScreen from './screens/GroupScreen';
 import LoginScreen from './screens/LoginScreen';
+import { getExponentPushToken, sendExpoNotifToken } from './src/notifications/notifTokenInit';
 
 const Stack = createNativeStackNavigator();
 
 export default function App(): React.ReactElement {
+  let isUserSignedIn = false;
```

Comment:
I was thinking we didn't really need to since we would be more so tracking the token right and seeing if it needs to be refreshed and everything.

Reply:
That makes sense. Maybe it could be renamed to something like `isTokenExpired` or maybe we could just use AuthSession's [`isTokenFresh`](https://usehooks-ts.com/react-hook/use-interval) to check if the user needs to log back in? This is probably for the next PR though

Reply:
I'm fine with that for now.  I'm just reluctant right now because I'm not sure exactly how the flow should work yet is all